Makefile

```diff
@@ -203,6 +203,10 @@ build-docs:
 		cp ~/output/$${path_prefix}/index.html ~/output ; \
 	done < versions ;
 
+view-docs: 
```

Comment:
Are we using `build-docs` somewhere? I guessed that we're using it for CI for deploying the docs or something.

Reply:
yes in the [ibc-docs repo](https://github.com/cosmos/ibc-docs/blob/2b57e3f4b786695a7302b3df2e1fc46d94e3e6c6/.github/workflows/deployment.yml#L27) - maybe we could move that make command to that repo